src/main/java/frc/robot/RobotContainer.java

```diff
@@ -57,6 +61,8 @@ private void configureBindings()
       ));
 
       m_primaryController.a().onTrue(new SeedFieldRelative(m_Drivetrain));
+      m_primaryController.b().whileTrue(new RunIntakeCommand(m_Intake, 0.3));
```

Comment:
Remove these controller binds. While these are good for testing, they shouldn't be added to our develop branch.